hw/ip/spi_host/dv/sva/spi_host_sva.core

```diff
@@ -10,6 +10,8 @@ filesets:
       - lowrisc:tlul:headers
       - lowrisc:fpv:csr_assert_gen
     files:
+      - ../../rtl/spi_host_reg_pkg.sv
```

Comment:
This isn't quite what you want, I don't think. Probably the right approach is to add a dependency on `lowrisc:ip:spi_host` (the core that includes this file). But then we get the same thing as `files_formal`, so can get rid of that fileset as well.